### PR TITLE
minor changes to ug

### PR DESCRIPTION
**Pull request by Eriksen2411**

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -166,6 +166,9 @@ To achieve the best possible experience, avoid adding excessively long text to C
     * 23.11.2021
     * 23112021
 * The year should be a non-negative number between 0000 and 9999.
+* The date should be a valid date.
+* When the day or month part of date only have 1 digit, a prefix 0 should be added.
```
Comment (lirc572) [inline docs/UserGuide.md]: only has one digit

File: docs/UserGuide.md [hunk 2]
```diff
@@ -639,10 +642,10 @@ Parameters:
...
 * [`PROJECT_INDEX`](#index) The index of the project.
 * [`EVENT_INDEX`](#index) The index of an event.
-* [`DESCRIPTION`](#description) The description of an event.
-* [`DATE`](#date) The date of the event, or the starting date of the event if the event repeats every week.
-* [`TIME`](#time) The time of the event.
-* [`REPEAT_WEEKLY`](#repeat_weekly) Indicates whether the event repeats every week.
+* [`DESCRIPTION`](#description) The updated description of an event.
+* [`DATE`](#date) The updated date of the event, or the starting date of the event if the event repeats every week.
+* [`TIME`](#time) The updated time of the event.
+* [`REPEAT_WEEKLY`](#repeat_weekly) Updates indicating whether the event repeats every week.
```
Comment (lirc572) [inline docs/UserGuide.md]: This line look a bit weird to me... I'm not sure what to change to tho...

File: docs/UserGuide.md [hunk 3]
```diff
@@ -809,11 +812,11 @@ Format: `updateC CONTACT_INDEX [n/NAME] [p/PHONE_NUMBER] [e/EMAIL] [a/ADDRESS] [
...
-* [`NAME`](#name) The name of a contact.
-* [`PHONE_NUMBER`](#phone_number) The phone number of a contact.
-* [`EMAIL`](#email) The email of a contact.
-* [`ADDRESS`](#address) The address of a contact.
-* [`TAG`](#tag) The tag associated with a contact.
+* [`NAME`](#name) The updated name of contact.
+* [`PHONE_NUMBER`](#phone_number) The updated phone number of contact.
+* [`EMAIL`](#email) The updated email of contact.
+* [`ADDRESS`](#address) The updated address of contact.
+* [`TAG`](#tag) The updated tag associated with contact.
```
Comment (lirc572) [inline docs/UserGuide.md]: For these lines, I think it's better to write `... of the contact`, to be consistent.

File: docs/UserGuide.md [hunk 4]
```diff
@@ -639,10 +642,10 @@ Parameters:
 
 * [`PROJECT_INDEX`](#index) The index of the project.
 * [`EVENT_INDEX`](#index) The index of an event.
-* [`DESCRIPTION`](#description) The description of an event.
```
Comment (lirc572) [inline docs/UserGuide.md:642]: of the event

**Review by lirc572 — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 4 above)
Comment (Eriksen2411) [reply docs/UserGuide.md:642]: done

File: docs/UserGuide.md (hunk 3 above)
Comment (Eriksen2411) [reply docs/UserGuide.md]: done

File: docs/UserGuide.md (hunk 2 above)
Comment (Eriksen2411) [reply docs/UserGuide.md]: yeah i think should keep

**Review by lirc572 — DISMISSED** [review]

File: docs/UserGuide.md [hunk 5]
```diff
@@ -166,6 +166,9 @@ To achieve the best possible experience, avoid adding excessively long text to C
     * 23.11.2021
     * 23112021
 * The year should be a non-negative number between 0000 and 9999.
+* The date should be a valid date.
+* When the day or month part of date only has 1 digit, a prefix 0 should be added.
+  e.g. if the day is 1, the day input should be `01` instead of just `1`.
```
Comment (vevek) [inline docs/UserGuide.md]: Phrase and change it to this
* Date should be a valid date (dd/MM/yyyy) where:
    * The Year Field (yyyy) should be a non-negative number between 0000 and 9999.
    * The Month Field (MM) should be a non-negative number between 01 and 12.
    * The Day Field (dd) should be a non-negative number between 00 and 31 (may be less than 31 depending on the number of days in the month).

File: docs/UserGuide.md [hunk 6]
```diff
@@ -724,8 +727,8 @@ Parameters:
 
 * [`PROJECT_INDEX`](#index) The index of the project.
 * [`GROUPMATE_INDEX`](#index) The index of the groupmate.
-* [`NAME`](#name) The name of the groupmate.
-* [`ROLE`](#role) The role of the groupmate.
+* [`NAME`](#name) The updated name of the groupmate.
+* [`ROLE`](#role) The updated role of the groupmate.
```
Comment (vevek) [inline docs/UserGuide.md]: role change to roles.

**Review by vevek — CHANGES_REQUESTED** [review]
I added some suggested changes!

**Review by vevek — APPROVED** [review]
LGTM Thanks!

**Pull request merged**